packages/node_modules/@ciscospark/recipe-private-web-client/src/index.js

```diff
@@ -18,6 +18,7 @@ import '@ciscospark/internal-plugin-support';
 import '@ciscospark/internal-plugin-team';
 import '@ciscospark/internal-plugin-user';
 import '@ciscospark/internal-plugin-wdm';
+import '@ciscospark/plugin-people';
```

Comment:
Do you need to put the showAllTypes config in here or can that be done in the web client?

Reply:
Good point. I think I should go ahead and do it here.

Reply:
Actually, we're already configuring the logger in web-client, so to be consistent, I guess I'll go ahead and do the configuration for people plugin there as well.

Reply:
FYI, I do have a local branch for web-client ready to go once this is merged in that I have set the config for. Verified everything gets sent as expected.